docs/archive-data.md

```diff
@@ -4,99 +4,75 @@ title: How to archive data
-
-There are two main reasons you may want to keep data after the retention period has past:
+There are at least two reasons you may want to keep data after the retention period has passed:
 
 1. **Compliance**: For legal reasons, data may need to be stored for a long period of time.
 2. **Debugging**: Older data may help with debugging.
 
-## Archival components
+:::note
 
-Archival consists of four main components.
+Archival is not supported when running Temporal via docker-compose.
 
-1. **Archival feature configuration**: Archival is controlled by the [`archival` configuration](https://github.com/temporalio/temporal/blob/master/config/development.yaml#L81) in the `config/development.yaml` file.
-2. **Archive Provider**: Location where the data is archived to. Commonly used providers are S3, GCloud, Kafka, and the local file system.
-3. **Archiver**: System component which archives and retrieves data. An Archiver is directly correlated to a single Archive Provider. 
-4. **URI**: Specifies to the Namespace which Archiver Provider and Archiver should be used. The schema of the URI identifies which Archiver and Provider to use. Where the URI points to identifies the archive location.
+Archival is disabled by default when installing the system manually and when deploying via [helm charts](https://github.com/temporalio/helm-charts/blob/master/templates/server-configmap.yaml#L176), but can be enabled in the [config](https://github.com/temporalio/temporal/blob/master/config/development.yaml).
 
-## Archival setup    
+Archival only supports Event Histories. You may notice some boilerplate infrastructure to support archiving visibility records. Visibility archival is not yet supported and visibility records are deleted after the Workflow retention period.
 
-To setup Archival, there are a few things you will need to do:
-1. Setup archive providers. 
-2. Manage Archival related configurations for the Temporal server.
-3. Create a Namespace that enables Archival and is registered with an Archival URI.
+:::
 
-### Archival providers
+## Setup
 
-Temporal supports several archive providers (and archivers) out of the box as well as the option to create and use your own.
+Archival consists of four elements.
 
-To archive data with a specific provider, Temporal must have a corresponding archiver component installed. [Currently supported archive providers](https://github.com/temporalio/temporal/tree/master/common/archiver) include the local filesystem (of whatever host Temporal is currently running on), Google Cloud, and S3.
+1. **Configuration**: Archival is controlled by the [server configuration](https://github.com/temporalio/temporal/blob/master/config/development.yaml#L81) (i.e. the `config/development.yaml` file).
+2. **Provider**: Location where the data should be archived. Supported providers are S3, GCloud, and the local file system.
+3. **Archiver**: System component which archives and retrieves data. Each archiver supports a specific provider. 
```

Comment:
Why is "archiver" lower case?

Reply:
So, I have been capitalizing "Archival" but not the individual elements.
Ironically "Archival" is abstract. There is no real "Archival" feature - there are several individual elements that together make archiving work... which we are calling "Archival".
We might need to revisit that - but for now, archiver, provider, and configuration are lowercase.